docs/pages/other-env.mdx

```diff
@@ -0,0 +1,193 @@
+# Tooling for other environments
+
+The main client-side feature of MUD is _synchronization_, having a local on-client copy of the up-to-date state of the `World` object.
+To support other environments, such as [Unity](https://unity.com/) or [Unreal Engine](https://www.unrealengine.com/), this is the main functionality that needs to be ported.
+
+The initial synchronization is best done using the [Indexer](indexer) to get the starting state.
+The Indexer is written in TypeScript, but it can be [accessed through HTTP](./indexer#using-http)
```

Comment:
The language the indexer is written in doesn't matter for the consumer.
There exist a library in Typescript to access the indexer, but -- as you mentioned -- under the hood it's JSON over HTTP using GET (for caching maybe?)